schedule/virt_autotest/slem-kubevirt-tests.yaml

```diff
@@ -0,0 +1,47 @@
+name:         slem-kubevirt-tests
+description:  >
+    Maintainer: Nan Zhang <nan.zhang@suse.com> qe-virt@suse.de
+    Run kubevirt tests on SLE Micro
+vars:
+    VIDEOMODE: text
+    SYSTEM_ROLE: kvm
+    OPT_KERNEL_PARAMS: hugepagesz=2M hugepages=64 max_loop=256
+schedule:
+    - '{{barrier_setup}}'
+    - '{{bootup_and_install}}'
+    - '{{kubevirt_tests}}'
+conditional_schedule:
+    barrier_setup:
+        TEST_SUITE_NAME:
+            slem-kubevirt-tests-server:
+                - virt_autotest/kubevirt_barriers
+    bootup_and_install:
+        WITH_SLEM_INSTALL:
+            1:
+                - '{{bootup}}'
+                - installation/welcome
+                - installation/scc_registration
+                - installation/ntp_config_settings
+                - installation/user_settings_root
+                - installation/resolve_dependency_issues
+                - installation/installation_overview
+                - installation/disable_grub_timeout
+                - installation/disable_grub_graphics
+                - installation/edit_optional_kernel_cmd_parameters
+                - installation/start_install
+                - installation/await_install
+                - installation/logs_from_installation_system
+                - installation/reboot_after_installation
+                - virt_autotest/login_console
+    bootup:
+        IPXE:
+            1:
+                - installation/ipxe_install
+            0:
+                - boot/boot_from_pxe
+    kubevirt_tests:
+        TEST_SUITE_NAME:
+            slem-kubevirt-tests-server:
+                - virt_autotest/kubevirt_tests_server
+            slem-kubevirt-tests-agent:
+                - virt_autotest/kubevirt_tests_agent
```

Comment:
I think you can try ```yq``` command to validate your yaml file format and syntax better.

Reply:
Thanks for advise! I got green for my yaml file by using this command.